termutils.py

```diff
@@ -0,0 +1,55 @@
+"""
+This module provides a context manager for setting the (Linux) terminal to raw mode.
+Author: https://github.com/Stefan-Code
```

Comment:
This project uses an AUTHORS.rst file for crediting contributors. Please use that instead.